test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/ItemViewDemo/ItemViewSample.cs

```diff
@@ -196,9 +216,7 @@ public void Activate()
             mItemView.PositionUsesPivotPoint = true;
             mItemView.ParentOrigin = Position.ParentOriginCenter;
             mItemView.PivotPoint = Position.PivotPointCenter;
```

Comment:
if you want to place mItemView at center of mContentView,
you can set FlexLayout instead of LinearLayout  to mItemLayout.
FlexLayout(){
  ItemsAlignment = FlexLayout.AlignmentType.Center,
 Justification = FlexLayout.FlexJustification.Center,
}

Reply:
Refined these codes. LinearLayout is enough after setting size for ItemView